apps/extension/src/ui/domains/Account/AllAccountsIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { UsersIcon } from "@talisman/theme/icons"
+import { classNames } from "@talismn/util"
+import { ReactNode } from "react"
+
+export const AllAccountsIcon = ({ className }: { className?: string }) => (
+  <AccountsIconContainer className={className}>
+    <UsersIcon className="text-primary w-full" />
+  </AccountsIconContainer>
+)
+
+export const AccountsIconContainer = ({
+  className,
+  children,
+}: {
+  className?: string
+  children?: ReactNode
+}) => (
+  <div
+    className={classNames(
+      "bg-grey-750 flex h-[1em] w-[1em] items-center justify-center rounded-full p-4",
```

Comment:
might want to have padding using `em` metric if the icon is meant to scale based on text size

Reply:
Nice catch